README.md

```diff
@@ -1,6 +1,6 @@
 #  WordPressKit for iOS
 
-[![CircleCI](https://circleci.com/gh/wordpress-mobile/WordPressKit-iOS.svg?style=svg)](https://circleci.com/gh/wordpress-mobile/WordPressKit-iOS)
+[![Build status](https://badge.buildkite.com/72552629875be91a9cc417a808e26f6130da69bc2bf91417bf.svg)](https://buildkite.com/wordpress-mobile/wordpresskit-ios)
```

Comment:
Same open discussion about badge [as in the other PR](https://github.com/wordpress-mobile/wpxmlrpc/pull/60#issuecomment-908191601).

No big deal and don't mind either way, so definitively not a blocker, though it could be nice to take the occasion to align all repos, either all with or all without a badge 🤷